plugins/modules/role_binding.py

```diff
@@ -72,24 +88,35 @@
 )
 
 
+def infer_role(params):
```

Comment:
Can we add two small test cases for this function? I know we already test it indirectly when performing other checks, but I would like to see those tests there just for the sake of completeness. Something like this should do:

```
@pytest.mark.parametrize("params,result", [
    (
        dict(role="my_role"),
        ("Role", "my_role"),
    ),
    (
        dict(cluster_role="my_cluster_role"),
        ("ClusterRole", "my_cluster_role"),
    ),
])
def test_infer_role(self, params, result):
    assert result == infer_role(params)
```

Reply:
Done. In the test cases, I'm also explicitly setting the other parameter to `None`, as this is the default case when we specify a parameter in module's argument spec and don't set it:
```
dict(cluster_role="a", role=None)
```